src/lib/processor.py

```diff
@@ -1,7 +1,10 @@
 import do
 import actions
 from exceptions import *
```

Comment:
BTW I don't think this does what you think it will do :)  Probably need to rename `exceptions` to something that doesn't conflict with a builtin module.